examples/search-tv-show/app/utils.server.tsx

```diff
@@ -0,0 +1,13 @@
+// Sometimes some modules don't work in the browser, Remix will generally be
```

Comment:
Can you delete this file? We don't need it.

Reply:
All done